databricks/koalas/series.py

```diff
@@ -80,6 +81,30 @@ def wrapper(self, *args):
     return wrapper
 
 
+def _pandas_column_op(f, spark_return_type, f_name=''):
```

Comment:
@icexelloss and @hjoo you two should synchronize to use the same infrastructure ... we probably shouldn't create two completely different infra for dt vs str.

Reply:
and btw I'm super excited about these two PRs! Will be a huge productivity boost.

Reply:
Yes we will sync up and use the same infrastructure for this.